contrib/python-bindings/include/manifold_wrapper.h

```diff
@@ -26,6 +30,43 @@ DEAL_II_NAMESPACE_OPEN
 
 namespace python
 {
+  template <int dim>
+  class FunctionWrapper : public Function<dim, double>
```

Comment:
I would create a separate file for this function since it is not tied to the `ManifoldWrapper` class and could be used elsewhere. Moving it later wouldn't be backward-compatible.